python_modules/dagster/dagster/_core/telemetry.py

```diff
@@ -76,8 +92,26 @@
     "BUILDKITE",  # https://buildkite.com/docs/pipelines/environment-variables
 }
 
+P = ParamSpec("P")
+T = TypeVar("T")
+T_Callable = TypeVar("T_Callable", bound=Callable[..., Any])
+
+TelemetryWrapperFn: TypeAlias = Callable[[T_Callable], T_Callable]
```

Comment:
Doesn't seem like this is used anywhere?

Reply:
good catch